coqpit/coqpit.py

```diff
@@ -115,9 +115,9 @@ def _default_value(x: Field):
     Returns:
         object: default value of the input Field.
     """
-    if x.default != MISSING:
```

Comment:
I'm not sure if it was a spelling mistake but if it wasn't I added the check for both MISSING object (_MISSING) and MISSING string(MISSING)